go.mod

```diff
@@ -344,6 +360,7 @@ require (
 	github.com/securego/gosec/v2 v2.22.2 // indirect
 	github.com/sergi/go-diff v1.3.2-0.20230802210424-5b0b94c5c0d3 // indirect
 	github.com/shirou/gopsutil/v3 v3.24.5 // indirect
```

Comment:
ps I think when envoy gateway gets bumped, it will include new func-e and hopefully kill the old gopsutil dip